identity/src/main/java/com/stripe/android/identity/IdentityVerificationSheet.kt

```diff
@@ -92,5 +96,31 @@ interface IdentityVerificationSheet {
             identityVerificationCallback: IdentityVerificationCallback
         ): IdentityVerificationSheet =
             StripeIdentityVerificationSheet(from, configuration, identityVerificationCallback)
+
+        /**
+         * Creates a [IdentityVerificationSheet] instance in a [Composable]. Which would be
+         * recreated if [configuration] or [identityVerificationCallback] changed.
+         *
+         * This API registers an [ActivityResultLauncher] into LocalContext.current and notifies its
+         * result to [identityVerificationCallback].
+         */
```

Comment:
I think it's worth adding a comment about recomposition, like we added on all other similar methods:
https://github.com/stripe/stripe-android/blob/master/payments-core/src/main/java/com/stripe/android/payments/paymentlauncher/PaymentLauncher.kt#L100-L108

> The IdentityVerificationSheet created is remembered across recompositions. Recomposition will
>  always return the value produced by composition.